internal/serverless/client.go

```diff
@@ -57,7 +57,6 @@ func NewClient(opts ...ClientOption) (*Client, error) {
 	if host != "" {
 		c.host = host
 	}
-	logger.Debugf("Using Elastic Cloud URL: %s", c.host)
```

Comment:
I think this message is not needed. WDYT ?
Another option is move that message to Trace log level.